src/backend/joanie/settings.py

```diff
@@ -228,6 +230,16 @@ class Base(Configuration):
     }
 
     CURRENCY = values.Value(("EUR", "\N{euro sign}"), environ_name="CURRENCY")
+    MARION_DOCUMENT_ISSUER_CHOICES_CLASS = "howard.defaults.DocumentIssuerChoices"
+    VAT = values.Value(20, environ_name="VAT")
+    INVOICE_COMPANY_CONTEXT = values.Value(
+        """
+        SARL dont le capital est de XXXXXX Euros<br>
+        RCS Paris XXX XXX XXX - SIRET XXX XXX XXX XXXXX - APE XXXXX<br>
+        Numéro de TVA XXXXXXXXX<br>
+        Siège social : 10 rue Stine, 75001 Paris""",
```

Comment:
Default value in English?

Reply:
Elsewhere we use english for sample code except if we need to test behiavor with accentuated text.